fundamentals/fundamentals-bio/src/test/scala/izumi/fundamentals/bio/test/BIOSyntaxTest.scala

```diff
@@ -1,12 +1,52 @@
 package izumi.fundamentals.bio.test
 
-import izumi.functional.bio.{BIO, BIOArrow, BIOAsk, BIOBifunctor3, BIOFork, BIOFork3, BIOFunctor, BIOLocal, BIOMonad, BIOMonad3, BIOMonadAsk, BIOMonadError, BIOPrimitives, BIOPrimitives3, BIOProfunctor, BIOTemporal, BIOTemporal3, F}
+import izumi.functional.bio.{BIO, BIOArrow, BIOAsk, BIOAsync, BIOBifunctor3, BIOFork, BIOFork3, BIOFunctor, BIOLocal, BIOMonad, BIOMonad3, BIOMonadAsk, BIOMonadError, BIOParallel, BIOPrimitives, BIOPrimitives3, BIOProfunctor, BIOTemporal, BIOTemporal3, F}
 import org.scalatest.wordspec.AnyWordSpec
 
 import scala.concurrent.duration._
+import izumi.functional.bio.BIOParallel3
+import izumi.functional.bio.BIOAsync3
 
 class BIOSyntaxTest extends AnyWordSpec {
 
+  "BIOParallel.zipPar/zipParLeft/zipParRight/zipWithPar is callable" in {
+    def x[F[+_, +_]: BIOParallel](a: F[Nothing, Unit], b: F[Nothing, Unit]) = {
+      a zipPar b
+      a zipParLeft b
+      a zipParRight b
+      a.zipWithPar(b)((a, b) => (a, b))
+    }
+
+    x[zio.IO](zio.UIO.succeed(()), zio.UIO.succeed(()))
+  }
+
+  "BIOParallel3.zipPar/zipParLeft/zipParRight/zipWithPar is callable" in {
+    def x[F[-_, +_, +_]: BIOParallel3](a: F[Any, Nothing, Unit], b: F[Any, Nothing, Unit]) = {
+      a zipPar b
+      a zipParLeft b
+      a zipParRight b
+      a.zipWithPar(b)((a, b) => (a, b))
+    }
+
+    x[zio.ZIO](zio.UIO.succeed(()), zio.UIO.succeed(()))
+  }
+
+  "BIOAsync.race is callable" in {
```

Comment:
@Adriani277 BIOParallel ops should also be available when _only_ BIOAsync is visible, the easiest way to do it would be to just copypaste them down into BIOAsync(3)Ops (as is done for BIOMonad ops)